tests/ceph_installer/test_ansible.py

```diff
@@ -169,6 +186,26 @@ def run(ceph_cluster, **kw):
         LOG.error("Failed during deployment")
         return rc
 
+    # Check logs has generated after running the deploy playbook
+    if custom_log:
```

Comment:
@adityaramteke as `custom_log: true`, this condition will get satisfied and no code will be executed aftet this if block. It will affect current modified test as well. Please confirm if this is intended.

Reply:
@vamahaja, yes as per the test if the condition is not satisfied after deployment no followed code will get executed and it's intended.